lib/WebDriver/AbstractWebDriver.php

```diff
@@ -122,15 +124,30 @@ protected function curl($requestMethod, $command, $parameters = null, $extraOpti
 
         list($rawResult, $info) = ServiceFactory::getInstance()->getService('service.curl')->execute($requestMethod, $url, $parameters, $extraOptions);
 
+        // According to the spec:
```

Comment:
I'm sure this huge comment can be shortened like: `According to <link here> all 4xx codes are considered as an error`.